bats/tests/helpers/utils.bash

```diff
@@ -38,6 +38,13 @@ validate_enum() {
     fatal "$var=${!var} is not a valid setting; select from [$*]"
 }
 
+validate_semver() {
```

Comment:
Since this is less obvious (`$1` is indirect), it would be good to have a doc-comment on what this does and how to use it.